src/PointwiseFunctions/Hydro/InitialData/IrrotationalBns.cpp

```diff
@@ -112,48 +112,45 @@ Scalar<DataType> enthalpy_density_squared(
 template <typename DataType>
 void spatial_rotational_killing_vector(
     const gsl::not_null<tnsr::I<DataType, 3>*> result,
-    const tnsr::I<DataType, 3>& x,
-    const Scalar<DataType>& local_angular_velocity_around_z,
+    const tnsr::I<DataType, 3>& x, const double orbital_angular_velocity,
     const Scalar<DataType>& sqrt_det_spatial_metric) {
   // Cross product involves volume element in arbitrary coordinates
   set_number_of_grid_points(result, sqrt_det_spatial_metric);
-  get<0>(*result) = -get(sqrt_det_spatial_metric) * get<1>(x) *
-                    get(local_angular_velocity_around_z);
-  get<1>(*result) = get(sqrt_det_spatial_metric) * get<0>(x) *
-                    get(local_angular_velocity_around_z);
+  get<0>(*result) =
+      -get(sqrt_det_spatial_metric) * get<1>(x) * orbital_angular_velocity;
+  get<1>(*result) =
+      get(sqrt_det_spatial_metric) * get<0>(x) * orbital_angular_velocity;
+  ;
   get<2>(*result) = 0.0;
 }
 
 template <typename DataType>
 tnsr::I<DataType, 3> spatial_rotational_killing_vector(
-    const tnsr::I<DataType, 3>& x,
-    const Scalar<DataType>& local_angular_velocity_around_z,
+    const tnsr::I<DataType, 3>& x, const double orbital_angular_velocity,
     const Scalar<DataType>& sqrt_det_spatial_metric) {
   tnsr::I<DataType, 3> buffer{};
   spatial_rotational_killing_vector(make_not_null(&buffer), x,
-                                    local_angular_velocity_around_z,
+                                    orbital_angular_velocity,
                                     sqrt_det_spatial_metric);
   return buffer;
 }
 
 template <typename DataType>
 void derivative_spatial_rotational_killing_vector(
     const gsl::not_null<tnsr::iJ<DataType, 3>*> result,
-    const tnsr::I<DataType, 3>& x,
-    const Scalar<DataType>& /*local_angular_velocity_around_z*/,
+    const tnsr::I<DataType, 3>& x, const double /*orbital_angular_velocity*/,
     const Scalar<DataType>& /*sqrt_det_spatial_metric*/) {
   set_number_of_grid_points(result, x);
   std::fill(result->begin(), result->end(), 0.0);
```

Comment:
Is this a placeholder? This shouldn't be zero, right?

Reply:
Er yeah, this was a placeholder, but now all we care about is the divergence, (i.e. trace of the derivative) which is zero.